typescript/phaser-arcade-slopes.d.ts

```diff
@@ -0,0 +1,259 @@
+declare module Phaser {
+	interface Game {
+		slopes:Phaser.Plugin.ArcadeSlopes;
+	}
+
+	module Physics.Arcade {
+		interface Body {
+			slopes:Phaser.Plugin.ArcadeSlopes;
+		}
+	}
+
+	module Plugin {
+		class ArcadeSlopes extends Phaser.Plugin {
+			static SAT:string;
+			static METROID:string;
+			enable(obj:Phaser.Sprite | Phaser.Group):void;
+			enableBody(body:Phaser.Physics.Arcade.Body):void;
+			convertTilemap(map:Phaser.Tilemap, layer:number | string | Phaser.TilemapLayer, slopeMap:Object):Phaser.Tilemap;
+			convertTilemapLayer(layer:Phaser.TilemapLayer, slopeMap:Object):Phaser.TilemapLayer;
+			collide(i:number, body:Phaser.Physics.Arcade.Body, tile:Phaser.Tile, overlapOnly:boolean):boolean;
+		}
+
+		module ArcadeSlopes {
+
+
+			class Facade {
+				factory:Phaser.Plugin.ArcadeSlopes.TileSlopeFactory;
+				solvers:Object;
+				defaultSover:number;
+				enable(obj:Phaser.Sprite | Phaser.Group):void;
+				enableBody(body:Phaser.Physics.Arcade.Body):void;
+				convertTilemap(map:Phaser.Tilemap, layer:number | string | Phaser.TilemapLayer, slopeMap:Object):Phaser.Tilemap;
+				convertTilemapLayer(layer:Phaser.TilemapLayer, slopeMap:Object):Phaser.TilemapLayer;
+				collide(i:number, body:Phaser.Physics.Arcade.Body, tile:Phaser.Tile, overlapOnly:boolean):boolean;
```

Comment:
I'm not following what is wrong.  All three of those classes have identical collide definitions in the source:

https://github.com/hexus/phaser-arcade-slopes/blob/master/src/ArcadeSlopes.js#L196
https://github.com/hexus/phaser-arcade-slopes/blob/master/src/ArcadeSlopes/SatSolver.js#L493
https://github.com/hexus/phaser-arcade-slopes/blob/master/src/ArcadeSlopes/Facade.js#L145

Are you referencing the i:number part?  If that's the case, Type/Javascript don't have an integer type.